swagger.json

```diff
@@ -427,7 +427,7 @@
         }
       }
     },
-    "/case/{caseIdentifier}/files-loaded-into-vidarr-archival": {
+    "/case/{caseIdentifier}/sent-to-vidarr-archival": {
```

Comment:
The URL now says 'sent to' vidarr archival, but the description says they were loaded. Is it possible to send to vidarr archival then have the loading process fail? What would be returned in that case?

Reply:
There's no Nabu code that talks to Vidarr, so it's the responsibility of the client/calling script to check that the `/load` request was successful before writing back to this Nabu endpoint. Here's the [current script that writes back to this endpoint ](https://bitbucket.oicr.on.ca/projects/GSI/repos/infrastructure/browse/bin/copy-files-to-vidarr-archival#310) after copying files and then loading metadata to vidarr-archival.